sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchIndexingBufferedSenderOptions.java

```diff
@@ -198,7 +199,8 @@ public Duration getRetryDelay() {
      *
      * @param maxRetryDelay The maximum duration requests will delay when the service is throttling.
      * @return The updated SearchIndexingBufferedSenderOptions object.
-     * @throws IllegalArgumentException If {@code maxRetryDelay.isNegative()} or {@code maxRetryDelay.isZero()} is true.
+     * @throws IllegalArgumentException If {@code maxRetryDelay.isNegative()} or {@code maxRetryDelay.isZero()} is
```

Comment:
Also document what happens if the user tries to set max delay to `null`.

Reply:
Currently null will revert back to the default, instead should it throw?